src/_tutorials/language/streams.md

```diff
@@ -166,9 +164,7 @@ src="{{site.custom.dartpadx.embed-inline-prefix}}?id=df7c1168a5c6b20fda2a76d6ff3
 </iframe>
 
 <aside class="alert alert-info" markdown="1">
-**Note:**
-Click run {% asset red-run.png alt="" %}
-to see the result in the **Console output**.
+**Note:** Click **Run** to see the result in the **Console**.
```

Comment:
@kwalrath just a heads up I also changed "the Console output" to just "the Console" since we changed that label in DartPad.